gulpfile.js

```diff
@@ -321,6 +321,29 @@ gulp.task('browser-sync', function() {
   });
 });
 
+/**
+ * Make sure that core client plugins are there
+ * 
+ * [submodule "src/plugins/qplotly"]     <-- https://github.com/g3w-suite/g3w-client-plugin-qplotly.git
+ * [submodule "src/plugins/qtimeseries"] <-- https://github.com/g3w-suite/g3w-client-plugin-qtimeseries.git
+ * [submodule "src/plugins/editing"]     <-- https://github.com/g3w-suite/g3w-client-plugin-editing.git
+ */
+gulp.task('clone:default_plugins', function() {
+  const { execSync } = require('child_process');
+  return new Promise(async done => {
+    for (const pluginName of ['editing', 'qplotly', 'qtimeseries']) {
+      if (!fs.existsSync(`${g3w.pluginsFolder}/${pluginName}/.git`)) {
+        execSync(`git clone https://github.com/g3w-suite/g3w-client-plugin-${pluginName}.git ${g3w.pluginsFolder}/${pluginName}`);
+      }
+      /** @TODO understand why the terminal hangs here (now you must run this 3 times to make it work) */
+      // if (!fs.existsSync(`${g3w.pluginsFolder}/${pluginName}/plugin.js`)) {
+      //   execSync(`npx gulp --gulpfile ${g3w.pluginsFolder}/${pluginName}/gulpfile.js browserify`);  
+      // }
```

Comment:
Probably somehow related to:

- the implicit directory change (cwd) due to the `--gulpfile` parameter
- the use of the `existsSync` command which does not throws any errors?

Reply:
Solved by: https://github.com/g3w-suite/g3w-client/pull/309/commits/5f3858de96d328607650d174bc4b81d252df02c2